src/components/Journal/Journal.js

```diff
@@ -15,11 +14,16 @@ const Journal = () => {
   }, [])
 
   return (
-    <div className='journal'>
-      {journal.length ?
-      journal.sort((a, b) => new Date(b.date) - new Date(a.date)).map(entry => <Entry key={entry.id} {...entry}/>)
-      : <p className='no-entries'>Search an album to add to your journal</p>}
```

Comment:
it looks like removing the `no-entries` p-tag is causing the test to fail - did you intend to remove this in your edit?

Reply:
Yes, to match the other headers for the same conditional test. I'll adjust the test